{
    "title" : "Authentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source",
    "uri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L7nSCAS/language:en_US",
    "printableUri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L7nSCAS/language:en_US",
    "clickUri" : "https://na61.salesforce.com/kA132000000L7nS",
    "uniqueId" : "42.13819$http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L7nSCAS/language:en_US",
    "excerpt" : "Authentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source. ... Error code: WEB_AUTHENTICATION_ERROR ... Cause ......",
    "firstSentences" : null,
    "summary" : null,
    "flags" : "HasHtmlVersion",
    "hasHtmlVersion" : true,
    "hasMobileHtmlVersion" : false,
    "score" : 1367,
    "percentScore" : 83.19566,
    "rankingInfo" : null,
    "rating" : 3.0,
    "isTopResult" : false,
    "isRecommendation" : false,
    "titleHighlights" : [ ],
    "firstSentencesHighlights" : [ ],
    "excerptHighlights" : [ ],
    "printableUriHighlights" : [ ],
    "summaryHighlights" : [ ],
    "parentResult" : null,
    "childResults" : [ ],
    "totalNumberOfChildResults" : 0,
    "raw" : {
      "systitle" : "Authentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source",
      "sysurihash" : "ryuQZzS79sbIffu5",
      "urihash" : "ryuQZzS79sbIffu5",
      "parents" : "<?xml version=\"1.0\" encoding=\"utf-16\"?><parents><parent name=\"Organization\" uri=\"https://na61.salesforce.com/home/home.jsp\" /><parent name=\"KB_Article__ka\" uri=\"http://www.salesforce.com/org:organization/articletype:KB_Article\" /><parent name=\"Authentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source\" uri=\"https://na61.salesforce.com/kA132000000L7nS\" /></parents>",
      "sfdcarticletype" : "How To/Question",
      "sfresolutionc" : "In this particular case, the issue was the following:<br><br>- The crawling address was: https://insidetyler.tylertech.com/<br><br>- When we try to crawl &quot;https://insidetyler.tylertech.com/&quot;, we&#39;re redirected to: &quot;https://insidetyler.tylertech.com/login?returnurl=%2f&quot; which redirect us to: <br>&quot;https://insidetyler.tylertech.com/DesktopModules/GS_ADProAuthentication/WinAuthSignIn.aspx?referer=%2flogin%3freturnurl%3d%252f&quot;<br><br>It&#39;s the last page which deal with the client (Tyler) NTLM authentication <br><br>So in order to crawl this WEB page properly, this is what needed to be done:<br><br>1) Enter the NTLM authentication page as crawling address:<br><br>https://insidetyler.tylertech.com/DesktopModules/GS_ADProAuthentication/WinAuthSignIn.aspx?referer=%2f<br><br>2) Two options:<br><br>- Either you disable  Robots.txt because in this case they set &quot;DesktopModules&quot; to &quot;Disallow&quot;, see: https://insidetyler.tylertech.com/robots.txt<br><br>- Or in the robots.txt (https://insidetyler.tylertech.com/robots.txt), add a Coveo User-agent &quot;CoveoBot&quot;, with the same rules as the other &quot;user-agents&quot;, except that it&#39;ll accept &quot;DesktopModules&quot;:<br> \n<pre class=\"ckeditor_codeblock\">\nUser-agent: CoveoBot\nDisallow: /*/ctl/        # CoveoBot permits *\nDisallow: /admin/\nDisallow: /App_Browsers/\nDisallow: /App_Code/\nDisallow: /App_Data/\nDisallow: /App_GlobalResources/\nDisallow: /bin/\nDisallow: /Components/\nDisallow: /Config/\nDisallow: /contest/\nDisallow: /controls/\nDisallow: /Documentation/\nDisallow: /HttpModules/\nDisallow: /images/\nDisallow: /Install/\nDisallow: /js/\nDisallow: /Portals/\nDisallow: /Providers/\nDisallow: /Resources/ContentRotator/\nDisallow: /Resources/ControlPanel/\nDisallow: /Resources/Dashboard/\nDisallow: /Resources/FeedBrowser/\nDisallow: /Resources/OpenForceAd/\nDisallow: /Resources/Search/\nDisallow: /Resources/Shared/\nDisallow: /Resources/SkinWidgets/\nDisallow: /Resources/TabStrip/\nDisallow: /Resources/Widgets/\nDisallow: /Activity-Feed/userId/    # Do not index user profiles</pre>\n<br> ",
      "sysuri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L7nSCAS/language:en_US",
      "sysprintableuri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L7nSCAS/language:en_US",
      "sflastpublisheddate" : 1481319062000,
      "systransactionid" : 161138,
      "sfurlname" : "1923",
      "sfsystemmodstamp" : 1517053488000,
      "sfisvisibleinapp" : "true",
      "sfcoveoversionc" : "Cloud Platform V2.0",
      "sysconcepts" : "authentication ; Error code ; Coveo Cloud ; client ; crawling address ; scrapper connector ; done ; Tyler",
      "sfkbversionnumber" : "2",
      "concepts" : "authentication ; Error code ; Coveo Cloud ; client ; crawling address ; scrapper connector ; done ; Tyler",
      "printableuri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L7nSCAS/language:en_US",
      "documenttype" : "article",
      "sfkbarticlekavid" : "ka132000000LJEeAAO",
      "sysindexeddate" : 1518187518000,
      "sfknowledgearticleid" : "kA132000000L7nSCAS",
      "sfcreatedbyid" : "00560000002MKBSAA4",
      "sfconnectorc" : "N/A",
      "permanentid" : "392ccc303c3b4a7d20aeb61b2733c1f1d0d8b9c24e4ba29bba7ab9505220",
      "syslanguage" : [ "English" ],
      "transactionid" : 161138,
      "sfisvisibleinprm" : "true",
      "title" : "Authentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source",
      "sfversionnumber" : 2,
      "sfarticlenumber" : "000001923",
      "date" : 1517053488000,
      "objecttype" : "KB Article",
      "allmetadatavalues" : "{\"kav_lastmodifiedby.attribute.type\": [\"User\"], \"coveo_short_id\": [\"kA132000000L7nS\"], \"sflastmodifiedbyname\": [\"Frederic Cote\"], \"kav_coveo_version__c__html_stripped\": [\"Cloud Platform V2.0\"], \"fileextension\": [\"\"], \"sfdcexpandedproduct\": [\"All;All|Cloud Platform\"], \"kav_createdby.attribute.url\": [\"/services/data/v34.0/sobjects/User/00560000002MKBSAA4\"], \"sflanguage\": [\"en_US\"], \"kav_createdby.attribute.type\": [\"User\"], \"kav_createdby.id\": [\"00560000002MKBSAA4\"], \"kav_language\": [\"en_US\"], \"sfsitecoreversionc\": [\"N/A\"], \"sourcetype\": [\"Salesforce\"], \"title\": [\"Authentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source\"], \"source\": [\"Salesforce Public Items\"], \"kav_owner.attribute.url\": [\"/services/data/v34.0/sobjects/User/00560000002MKBSAA4\"], \"parents\": [\"<?xml version=\\\"1.0\\\" encoding=\\\"utf-16\\\"?><parents><parent name=\\\"Organization\\\" uri=\\\"https://na61.salesforce.com/home/home.jsp\\\" /><parent name=\\\"KB_Article__ka\\\" uri=\\\"http://www.salesforce.com/org:organization/articletype:KB_Article\\\" /><parent name=\\\"Authentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source\\\" uri=\\\"https://na61.salesforce.com/kA132000000L7nS\\\" /></parents>\"], \"sfisvisibleinprm\": [true], \"kav_title\": [\"Authentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source\"], \"kav_isvisibleincsp\": [true], \"hashtmlversion\": [true], \"kav_versionnumber\": [2], \"sfcreatedbyname\": [\"Karel Mpungi\"], \"sflastpublisheddate\": [1481319062], \"objecttypelabel\": [\"KB Article\"], \"kav_cause__c__html_stripped\": [\"The client was unable to crawl a WEB site using the WEB scrapper connector in Cloud V2. \\n \\n He was getting the following error: \\n \\u00a0\\n \\nAuthentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source.\\n\\n\\nError code: WEB_AUTHENTICATION_ERROR \\n \\n \"], \"originalhtmlcharset\": [1200], \"coveo_connector_hasbinarydata\": [false], \"kav_lastpublisheddate\": [1481319062], \"sfsystemmodstamp\": [1517053488], \"ka_attribute.type\": [\"KB_Article__ka\"], \"sfdcproduct\": [\"Cloud Platform\"], \"kav_resolution__c__html_stripped\": [\"In this particular case, the issue was the following: \\n \\n - The crawling address was: https://insidetyler.tylertech.com/ \\n \\n - When we try to crawl \\\"https://insidetyler.tylertech.com/\\\", we're redirected to: \\\"https://insidetyler.tylertech.com/login?returnurl=%2f\\\" which redirect us to:\\u00a0 \\n \\\"https://insidetyler.tylertech.com/DesktopModules/GS_ADProAuthentication/WinAuthSignIn.aspx?referer=%2flogin%3freturnurl%3d%252f\\\" \\n \\n It's the last page which deal with the client (Tyler) NTLM authentication\\u00a0 \\n \\n So in order to crawl this WEB page properly, this is what needed to be done: \\n \\n 1) Enter the NTLM authentication page as crawling address: \\n \\n https://insidetyler.tylertech.com/DesktopModules/GS_ADProAuthentication/WinAuthSignIn.aspx?referer=%2f \\n \\n 2) Two options: \\n \\n - Either you disable \\u00a0Robots.txt because in this case they set \\\"DesktopModules\\\" to \\\"Disallow\\\", see: https://insidetyler.tylertech.com/robots.txt \\n \\n - Or in the robots.txt (https://insidetyler.tylertech.com/robots.txt), add a Coveo User-agent \\\"CoveoBot\\\", with the same rules as the other \\\"user-agents\\\", except that it'll accept \\\"DesktopModules\\\": \\n \\u00a0\\n \\nUser-agent: CoveoBot\\nDisallow: /*/ctl/\\u00a0\\u00a0 \\u00a0\\u00a0\\u00a0 \\u00a0# CoveoBot permits *\\nDisallow: /admin/\\nDisallow: /App_Browsers/\\nDisallow: /App_Code/\\nDisallow: /App_Data/\\nDisallow: /App_GlobalResources/\\nDisallow: /bin/\\nDisallow: /Components/\\nDisallow: /Config/\\nDisallow: /contest/\\nDisallow: /controls/\\nDisallow: /Documentation/\\nDisallow: /HttpModules/\\nDisallow: /images/\\nDisallow: /Install/\\nDisallow: /js/\\nDisallow: /Portals/\\nDisallow: /Providers/\\nDisallow: /Resources/ContentRotator/\\nDisallow: /Resources/ControlPanel/\\nDisallow: /Resources/Dashboard/\\nDisallow: /Resources/FeedBrowser/\\nDisallow: /Resources/OpenForceAd/\\nDisallow: /Resources/Search/\\nDisallow: /Resources/Shared/\\nDisallow: /Resources/SkinWidgets/\\nDisallow: /Resources/TabStrip/\\nDisallow: /Resources/Widgets/\\nDisallow: /Activity-Feed/userId/\\u00a0\\u00a0 \\u00a0# Do not index user profiles \\n \\n \\u00a0\"], \"clickableuri\": [\"https://na61.salesforce.com/kA132000000L7nS\"], \"sftitle\": [\"Authentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source\"], \"kav_owner.id\": [\"00560000002MKBSAA4\"], \"sfisvisibleinapp\": [true], \"ka_attribute.url\": [\"/services/data/v34.0/sobjects/KB_Article__ka/kA132000000L7nSCAS\"], \"sfdcarticletype\": [\"How To/Question\"], \"kav_cause__c\": [\"The client was unable to crawl a WEB site using the WEB scrapper connector in Cloud V2.<br><br>He was getting the following error:<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\nAuthentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source.\\n\\n\\nError code: WEB_AUTHENTICATION_ERROR</pre>\\n<br><img alt=\\\"User-added image\\\" height=\\\"82\\\" src=\\\"https://c.na61.content.force.com/servlet/rtaImage?eid=ka132000000LJEe&amp;feoid=00N32000002zy1O&amp;refid=0EM32000000VFm1\\\" width=\\\"500\\\"></img>\"], \"sfislatestversion\": [true], \"sfcausec\": [\"The client was unable to crawl a WEB site using the WEB scrapper connector in Cloud V2.<br><br>He was getting the following error:<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\nAuthentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source.\\n\\n\\nError code: WEB_AUTHENTICATION_ERROR</pre>\\n<br><img alt=\\\"User-added image\\\" height=\\\"82\\\" src=\\\"https://c.na61.content.force.com/servlet/rtaImage?eid=ka132000000LJEe&amp;feoid=00N32000002zy1O&amp;refid=0EM32000000VFm1\\\" width=\\\"500\\\"></img>\"], \"contenttype\": [\"binarydata\"], \"sfurlname\": [\"1923\"], \"ka_id\": [\"kA132000000L7nSCAS\"], \"kav_datacategoryselections.datacategoryname\": [\"How_To;Cloud_Platform\"], \"coveo_url\": [\"https://na61.salesforce.com/kA132000000L7nS\"], \"kav_urlname\": [\"1923\"], \"kav_dc_expanded_product\": [\"All;All|Cloud Platform\"], \"foldingcollection\": [\"2918cd8f8041db4fdded7dd414a33bb6ec96dad0f4023b08024051142f2b\"], \"body\": [\"<html>   <body>     <div style=\\\"background-color: #fff;border: 2px solid #BBC5CC;border-radius: 4px;float: left;font-family: lato;margin-left: 15px;margin-right: 15px;margin-bottom: 4%;padding: 10px;width: 60%;\\\">       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Description         </div>       </div>       <div style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;\\\"> This article explains the configuration that needs to be done when encountering the error below when crawling a WEB page with the WEB scrapper connector in Cloud V2<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\nAuthentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source.\\n\\nError code: WEB_AUTHENTICATION_ERROR</pre>\\n<br><img alt=\\\"User-added image\\\" height=\\\"82\\\" src=\\\"https://c.na61.content.force.com/servlet/rtaImage?eid=ka132000000LJEe&amp;feoid=00N32000002zy14&amp;refid=0EM32000000VFlw\\\" width=\\\"500\\\"></img></div>       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Environment         </div>       </div>       <br/>       <table>         <tr>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Product</td>           <td></td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Sitecore Version</td>           <td>N/A</td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Connector</td>           <td>N/A</td>         </tr>         <tr>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Published</td>           <td>1481319062</td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Version</td>           <td></td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Build</td>           <td></td>         </tr>       </table>       <br/>       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Cause         </div>       </div>       <div style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;\\\"> The client was unable to crawl a WEB site using the WEB scrapper connector in Cloud V2.<br><br>He was getting the following error:<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\nAuthentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source.\\n\\n\\nError code: WEB_AUTHENTICATION_ERROR</pre>\\n<br><img alt=\\\"User-added image\\\" height=\\\"82\\\" src=\\\"https://c.na61.content.force.com/servlet/rtaImage?eid=ka132000000LJEe&amp;feoid=00N32000002zy1O&amp;refid=0EM32000000VFm1\\\" width=\\\"500\\\"></img></div>       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Resolution         </div>       </div>       <div style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;\\\"> In this particular case, the issue was the following:<br><br>- The crawling address was: https://insidetyler.tylertech.com/<br><br>- When we try to crawl &quot;https://insidetyler.tylertech.com/&quot;, we&#39;re redirected to: &quot;https://insidetyler.tylertech.com/login?returnurl=%2f&quot; which redirect us to:\\u00a0<br>&quot;https://insidetyler.tylertech.com/DesktopModules/GS_ADProAuthentication/WinAuthSignIn.aspx?referer=%2flogin%3freturnurl%3d%252f&quot;<br><br>It&#39;s the last page which deal with the client (Tyler) NTLM authentication\\u00a0<br><br>So in order to crawl this WEB page properly, this is what needed to be done:<br><br>1) Enter the NTLM authentication page as crawling address:<br><br>https://insidetyler.tylertech.com/DesktopModules/GS_ADProAuthentication/WinAuthSignIn.aspx?referer=%2f<br><br>2) Two options:<br><br>- Either you disable \\u00a0Robots.txt because in this case they set &quot;DesktopModules&quot; to &quot;Disallow&quot;, see: https://insidetyler.tylertech.com/robots.txt<br><br>- Or in the robots.txt (https://insidetyler.tylertech.com/robots.txt), add a Coveo User-agent &quot;CoveoBot&quot;, with the same rules as the other &quot;user-agents&quot;, except that it&#39;ll accept &quot;DesktopModules&quot;:<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\nUser-agent: CoveoBot\\nDisallow: /*/ctl/\\u00a0\\u00a0 \\u00a0\\u00a0\\u00a0 \\u00a0# CoveoBot permits *\\nDisallow: /admin/\\nDisallow: /App_Browsers/\\nDisallow: /App_Code/\\nDisallow: /App_Data/\\nDisallow: /App_GlobalResources/\\nDisallow: /bin/\\nDisallow: /Components/\\nDisallow: /Config/\\nDisallow: /contest/\\nDisallow: /controls/\\nDisallow: /Documentation/\\nDisallow: /HttpModules/\\nDisallow: /images/\\nDisallow: /Install/\\nDisallow: /js/\\nDisallow: /Portals/\\nDisallow: /Providers/\\nDisallow: /Resources/ContentRotator/\\nDisallow: /Resources/ControlPanel/\\nDisallow: /Resources/Dashboard/\\nDisallow: /Resources/FeedBrowser/\\nDisallow: /Resources/OpenForceAd/\\nDisallow: /Resources/Search/\\nDisallow: /Resources/Shared/\\nDisallow: /Resources/SkinWidgets/\\nDisallow: /Resources/TabStrip/\\nDisallow: /Resources/Widgets/\\nDisallow: /Activity-Feed/userId/\\u00a0\\u00a0 \\u00a0# Do not index user profiles</pre>\\n<br>\\u00a0</div>     </div>   </body> </html>\"], \"sfpublishstatus\": [\"Online\"], \"sfcreatedbyid\": [\"00560000002MKBSAA4\"], \"kav_description__c__html_stripped\": [\"This article explains the configuration that needs to be done when encountering the error below when crawling a WEB page with the WEB scrapper connector in Cloud V2 \\n \\u00a0\\n \\nAuthentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source.\\n\\nError code: WEB_AUTHENTICATION_ERROR \\n \\n \"], \"kav_islatestversion\": [true], \"kav_knowledgearticleid\": [\"kA132000000L7nSCAS\"], \"sfknowledgearticleid\": [\"kA132000000L7nSCAS\"], \"sfownername\": [\"Karel Mpungi\"], \"kav_datacategoryselections.attribute.type\": [\"KB_Article__DataCategorySelection;KB_Article__DataCategorySelection\"], \"detectedtitle\": [\"Error code: WEB_AUTHENTICATION_ERROR\"], \"kav_createddate\": [1481318846], \"language\": [\"English\"], \"coveo_foldingchild\": [\"a0c4c2a1d4142c36a6c9503fa8891c5e1ee5897fddb96202939b9ef4b13a\"], \"sfownerid\": [\"00560000002MKBSAA4\"], \"kav_createdby.coveo_record_size\": [2], \"kav_systemmodstamp\": [1517053488], \"objecttype\": [\"KB Article\"], \"connectortype\": [\"Salesforce2\"], \"kav_coveo_record_size\": [4], \"originaluri\": [\"http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L7nSCAS/language:en_US\"], \"sfisvisibleincsp\": [true], \"convertertype\": [\"TYPE_HTML\"], \"filetype\": [\"SalesforceItem\"], \"kav_owner.name\": [\"Karel Mpungi\"], \"sfcoveoversionc\": [\"Cloud Platform V2.0\"], \"sfdcexpandedarticletype\": [\"All;All|How To/Question\"], \"objecttypelabelplural\": [\"KB Articles\"], \"sfcurrencyisocode\": [\"USD\"], \"sfisdeleted\": [false], \"extractedsize\": [5398], \"size\": [8350], \"kav_publishstatus\": [\"Online\"], \"kav_connector__c\": [\"N/A\"], \"generateexcerpt\": [true], \"sfdescriptionc\": [\"This article explains the configuration that needs to be done when encountering the error below when crawling a WEB page with the WEB scrapper connector in Cloud V2<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\nAuthentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source.\\n\\nError code: WEB_AUTHENTICATION_ERROR</pre>\\n<br><img alt=\\\"User-added image\\\" height=\\\"82\\\" src=\\\"https://c.na61.content.force.com/servlet/rtaImage?eid=ka132000000LJEe&amp;feoid=00N32000002zy14&amp;refid=0EM32000000VFlw\\\" width=\\\"500\\\"></img>\"], \"sfarticletype\": [\"KB_Article__kav\"], \"kav_articletype\": [\"KB_Article__kav\"], \"kav_lastmodifieddate\": [1481319062], \"documenttype\": [\"article\"], \"sfkbarticlekavid\": [\"ka132000000LJEeAAO\"], \"kav_currencyisocode\": [\"USD\"], \"foldingparent\": [\"a0c4c2a1d4142c36a6c9503fa8891c5e1ee5897fddb96202939b9ef4b13a\"], \"mappingtype\": [\"KB_Article\"], \"kav_isvisibleinpkb\": [true], \"kav_sourceid\": [\"50032000016vrqFAAQ\"], \"sfkbversionnumber\": [2], \"kav_isvisibleinapp\": [true], \"kav_resolution__c\": [\"In this particular case, the issue was the following:<br><br>- The crawling address was: https://insidetyler.tylertech.com/<br><br>- When we try to crawl &quot;https://insidetyler.tylertech.com/&quot;, we&#39;re redirected to: &quot;https://insidetyler.tylertech.com/login?returnurl=%2f&quot; which redirect us to:\\u00a0<br>&quot;https://insidetyler.tylertech.com/DesktopModules/GS_ADProAuthentication/WinAuthSignIn.aspx?referer=%2flogin%3freturnurl%3d%252f&quot;<br><br>It&#39;s the last page which deal with the client (Tyler) NTLM authentication\\u00a0<br><br>So in order to crawl this WEB page properly, this is what needed to be done:<br><br>1) Enter the NTLM authentication page as crawling address:<br><br>https://insidetyler.tylertech.com/DesktopModules/GS_ADProAuthentication/WinAuthSignIn.aspx?referer=%2f<br><br>2) Two options:<br><br>- Either you disable \\u00a0Robots.txt because in this case they set &quot;DesktopModules&quot; to &quot;Disallow&quot;, see: https://insidetyler.tylertech.com/robots.txt<br><br>- Or in the robots.txt (https://insidetyler.tylertech.com/robots.txt), add a Coveo User-agent &quot;CoveoBot&quot;, with the same rules as the other &quot;user-agents&quot;, except that it&#39;ll accept &quot;DesktopModules&quot;:<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\nUser-agent: CoveoBot\\nDisallow: /*/ctl/\\u00a0\\u00a0 \\u00a0\\u00a0\\u00a0 \\u00a0# CoveoBot permits *\\nDisallow: /admin/\\nDisallow: /App_Browsers/\\nDisallow: /App_Code/\\nDisallow: /App_Data/\\nDisallow: /App_GlobalResources/\\nDisallow: /bin/\\nDisallow: /Components/\\nDisallow: /Config/\\nDisallow: /contest/\\nDisallow: /controls/\\nDisallow: /Documentation/\\nDisallow: /HttpModules/\\nDisallow: /images/\\nDisallow: /Install/\\nDisallow: /js/\\nDisallow: /Portals/\\nDisallow: /Providers/\\nDisallow: /Resources/ContentRotator/\\nDisallow: /Resources/ControlPanel/\\nDisallow: /Resources/Dashboard/\\nDisallow: /Resources/FeedBrowser/\\nDisallow: /Resources/OpenForceAd/\\nDisallow: /Resources/Search/\\nDisallow: /Resources/Shared/\\nDisallow: /Resources/SkinWidgets/\\nDisallow: /Resources/TabStrip/\\nDisallow: /Resources/Widgets/\\nDisallow: /Activity-Feed/userId/\\u00a0\\u00a0 \\u00a0# Do not index user profiles</pre>\\n<br>\\u00a0\"], \"coveo_objecttype\": [\"KB_Article\"], \"printableuri\": [\"http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L7nSCAS/language:en_US\"], \"kav_datacategoryselections.attribute.url\": [\"/services/data/v34.0/sobjects/KB_Article__DataCategorySelection/02o32000000pst3AAA;/services/data/v34.0/sobjects/KB_Article__DataCategorySelection/02o32000000pst7AAA\"], \"kav_id\": [\"ka132000000LJEeAAO\"], \"sfresolutionc\": [\"In this particular case, the issue was the following:<br><br>- The crawling address was: https://insidetyler.tylertech.com/<br><br>- When we try to crawl &quot;https://insidetyler.tylertech.com/&quot;, we&#39;re redirected to: &quot;https://insidetyler.tylertech.com/login?returnurl=%2f&quot; which redirect us to:\\u00a0<br>&quot;https://insidetyler.tylertech.com/DesktopModules/GS_ADProAuthentication/WinAuthSignIn.aspx?referer=%2flogin%3freturnurl%3d%252f&quot;<br><br>It&#39;s the last page which deal with the client (Tyler) NTLM authentication\\u00a0<br><br>So in order to crawl this WEB page properly, this is what needed to be done:<br><br>1) Enter the NTLM authentication page as crawling address:<br><br>https://insidetyler.tylertech.com/DesktopModules/GS_ADProAuthentication/WinAuthSignIn.aspx?referer=%2f<br><br>2) Two options:<br><br>- Either you disable \\u00a0Robots.txt because in this case they set &quot;DesktopModules&quot; to &quot;Disallow&quot;, see: https://insidetyler.tylertech.com/robots.txt<br><br>- Or in the robots.txt (https://insidetyler.tylertech.com/robots.txt), add a Coveo User-agent &quot;CoveoBot&quot;, with the same rules as the other &quot;user-agents&quot;, except that it&#39;ll accept &quot;DesktopModules&quot;:<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\nUser-agent: CoveoBot\\nDisallow: /*/ctl/\\u00a0\\u00a0 \\u00a0\\u00a0\\u00a0 \\u00a0# CoveoBot permits *\\nDisallow: /admin/\\nDisallow: /App_Browsers/\\nDisallow: /App_Code/\\nDisallow: /App_Data/\\nDisallow: /App_GlobalResources/\\nDisallow: /bin/\\nDisallow: /Components/\\nDisallow: /Config/\\nDisallow: /contest/\\nDisallow: /controls/\\nDisallow: /Documentation/\\nDisallow: /HttpModules/\\nDisallow: /images/\\nDisallow: /Install/\\nDisallow: /js/\\nDisallow: /Portals/\\nDisallow: /Providers/\\nDisallow: /Resources/ContentRotator/\\nDisallow: /Resources/ControlPanel/\\nDisallow: /Resources/Dashboard/\\nDisallow: /Resources/FeedBrowser/\\nDisallow: /Resources/OpenForceAd/\\nDisallow: /Resources/Search/\\nDisallow: /Resources/Shared/\\nDisallow: /Resources/SkinWidgets/\\nDisallow: /Resources/TabStrip/\\nDisallow: /Resources/Widgets/\\nDisallow: /Activity-Feed/userId/\\u00a0\\u00a0 \\u00a0# Do not index user profiles</pre>\\n<br>\\u00a0\"], \"foldingchild\": [\"a0c4c2a1d4142c36a6c9503fa8891c5e1ee5897fddb96202939b9ef4b13a\"], \"sfcreateddate\": [1481318846], \"kav_sitecore_version__c\": [\"N/A\"], \"kav_owner.coveo_record_size\": [2], \"kav_coveo_effective_modstamp\": [1517053488], \"kav_summary__html_stripped\": [\"\"], \"kav_dc_article_type\": [\"How To/Question\"], \"sfconnectorc\": [\"N/A\"], \"kav_dc_product\": [\"Cloud Platform\"], \"ka_coveo_record_size\": [2], \"kav_dc_expanded_article_type\": [\"All;All|How To/Question\"], \"kav_datacategoryselections.coveo_record_size\": [\"2;2\"], \"concepts\": [\"authentication ; Error code ; Coveo Cloud ; client ; crawling address ; scrapper connector ; done ; Tyler\"], \"sfkbid\": [\"kA132000000L7nSCAS\"], \"kav_firstpublisheddate\": [1481053901], \"coveo_organization_id\": [\"00D3000000007r2\"], \"coveo_foldingcollection\": [\"2918cd8f8041db4fdded7dd414a33bb6ec96dad0f4023b08024051142f2b\"], \"sffirstpublisheddate\": [1481053901], \"sflastmodifieddate\": [1481319062], \"kav_coveo_version__c\": [\"Cloud Platform V2.0\"], \"kav_vendor_version__c__html_stripped\": [\"\"], \"detectedlanguage\": [1], \"kav_isvisibleinprm\": [true], \"kav_owner.attribute.type\": [\"Name\"], \"kav_lastmodifiedby.attribute.url\": [\"/services/data/v34.0/sobjects/User/00560000003q7JgAAI\"], \"sitename\": [\"Community\"], \"kav_createdby.name\": [\"Karel Mpungi\"], \"sfarticlenumber\": [\"000001923\"], \"kav_lastmodifiedby.coveo_record_size\": [2], \"sfisvisibleinpkb\": [true], \"kav_datacategoryselections.id\": [\"02o32000000pst3AAA;02o32000000pst7AAA\"], \"kav_description__c\": [\"This article explains the configuration that needs to be done when encountering the error below when crawling a WEB page with the WEB scrapper connector in Cloud V2<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\nAuthentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source.\\n\\nError code: WEB_AUTHENTICATION_ERROR</pre>\\n<br><img alt=\\\"User-added image\\\" height=\\\"82\\\" src=\\\"https://c.na61.content.force.com/servlet/rtaImage?eid=ka132000000LJEe&amp;feoid=00N32000002zy14&amp;refid=0EM32000000VFlw\\\" width=\\\"500\\\"></img>\"], \"kav_attribute.url\": [\"/services/data/v34.0/sobjects/KB_Article__kav/ka132000000LJEeAAO\"], \"kav_articlenumber__stripped\": [\"1923\"], \"sfversionnumber\": [2], \"kav_attribute.type\": [\"KB_Article__kav\"], \"kav_validationstatus\": [\"Reviewed\"], \"coveo_foldingparent\": [\"a0c4c2a1d4142c36a6c9503fa8891c5e1ee5897fddb96202939b9ef4b13a\"], \"collection\": [\"default\"], \"sfsourceid\": [\"50032000016vrqFAAQ\"], \"date\": [1517053488], \"kav_lastmodifiedby.id\": [\"00560000003q7JgAAI\"], \"kav_datacategoryselections.datacategorygroupname\": [\"Article_Type;Product\"], \"conversionstate\": [0], \"sfvalidationstatus\": [\"Reviewed\"], \"kav_isdeleted\": [false], \"sflastmodifiedbyid\": [\"00560000003q7JgAAI\"], \"kav_lastmodifiedby.name\": [\"Frederic Cote\"], \"kav_articlenumber\": [\"000001923\"], \"permanentid\": [\"392ccc303c3b4a7d20aeb61b2733c1f1d0d8b9c24e4ba29bba7ab9505220\"]}",
      "sflastmodifiedbyid" : "00560000003q7JgAAI",
      "syssfownerid" : "00560000002MKBSAA4",
      "sfsitecoreversionc" : "N/A",
      "sourcetype" : "Salesforce",
      "sysconnectortype" : "Salesforce2",
      "sftitle" : "Authentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source",
      "rowid" : 5492153,
      "syssfcreatedbyid" : "00560000002MKBSAA4",
      "sfdescriptionc" : "This article explains the configuration that needs to be done when encountering the error below when crawling a WEB page with the WEB scrapper connector in Cloud V2<br> \n<pre class=\"ckeditor_codeblock\">\nAuthentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source.\n\nError code: WEB_AUTHENTICATION_ERROR</pre>\n<br><img alt=\"User-added image\" height=\"82\" src=\"https://c.na61.content.force.com/servlet/rtaImage?eid=ka132000000LJEe&amp;feoid=00N32000002zy14&amp;refid=0EM32000000VFlw\" width=\"500\"></img>",
      "size" : 8350,
      "sfislatestversion" : "true",
      "sysdocumenttype" : "article",
      "sfcreateddate" : 1481318846000,
      "sfsourceid" : "50032000016vrqFAAQ",
      "clickableuri" : "https://na61.salesforce.com/kA132000000L7nS",
      "syssource" : "Salesforce Public Items",
      "sfownerid" : "00560000002MKBSAA4",
      "orderingid" : 212384990660370943,
      "sfpublishstatus" : "Online",
      "syssize" : 8350,
      "sysdate" : 1517053488000,
      "sflanguage" : "en_US",
      "sysparents" : "<?xml version=\"1.0\" encoding=\"utf-16\"?><parents><parent name=\"Organization\" uri=\"https://na61.salesforce.com/home/home.jsp\" /><parent name=\"KB_Article__ka\" uri=\"http://www.salesforce.com/org:organization/articletype:KB_Article\" /><parent name=\"Authentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source\" uri=\"https://na61.salesforce.com/kA132000000L7nS\" /></parents>",
      "sfdcexpandedarticletype" : "All;All|How To/Question",
      "sfcurrencyisocode" : "USD",
      "sfkbid" : "kA132000000L7nSCAS",
      "sfcausec" : "The client was unable to crawl a WEB site using the WEB scrapper connector in Cloud V2.<br><br>He was getting the following error:<br> \n<pre class=\"ckeditor_codeblock\">\nAuthentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source.\n\n\nError code: WEB_AUTHENTICATION_ERROR</pre>\n<br><img alt=\"User-added image\" height=\"82\" src=\"https://c.na61.content.force.com/servlet/rtaImage?eid=ka132000000LJEe&amp;feoid=00N32000002zy1O&amp;refid=0EM32000000VFm1\" width=\"500\"></img>",
      "sfdcexpandedproduct" : "All;All|Cloud Platform",
      "syssfcreateddate" : 1481318846000,
      "sflastmodifieddate" : 1481319062000,
      "sfisvisibleinpkb" : "true",
      "source" : "Salesforce Public Items",
      "sfisdeleted" : "false",
      "sfisvisibleincsp" : "true",
      "sfvalidationstatus" : "Reviewed",
      "sflastmodifiedbyname" : "Frederic Cote",
      "collection" : "default",
      "sfdcproduct" : [ "Cloud Platform" ],
      "syssourcetype" : "Salesforce",
      "sfarticletype" : "KB_Article__kav",
      "indexeddate" : 1518187518000,
      "connectortype" : "Salesforce2",
      "filetype" : "SalesforceItem",
      "sfcreatedbyname" : "Karel Mpungi",
      "sysclickableuri" : "https://na61.salesforce.com/kA132000000L7nS",
      "sysfiletype" : "SalesforceItem",
      "language" : [ "English" ],
      "sffirstpublisheddate" : 1481053901000,
      "sfownername" : "Karel Mpungi",
      "sitename" : "Community",
      "sysrowid" : 5492153,
      "uri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L7nSCAS/language:en_US",
      "syscollection" : "default"
    },
    "Title" : "Authentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source",
    "Uri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L7nSCAS/language:en_US",
    "PrintableUri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L7nSCAS/language:en_US",
    "ClickUri" : "https://na61.salesforce.com/kA132000000L7nS",
    "UniqueId" : "42.13819$http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L7nSCAS/language:en_US",
    "Excerpt" : "Authentication issue: Coveo Cloud is not able to authenticate to the website and consequently cannot perform any action regarding your source. ... Error code: WEB_AUTHENTICATION_ERROR ... Cause ......",
    "FirstSentences" : null
  }